web/src/compose_actions.ts

```diff
@@ -152,7 +175,15 @@ export function maybe_scroll_up_selected_message(opts) {
     }
 }
 
-export function fill_in_opts_from_current_narrowed_view(opts) {
+type ComposeActionsOpts = ComposeActionsStartOpts & {
+    topic: string;
+    private_message_recipient: string;
+    trigger: string;
+};
```

Comment:
Define `ComposeActionsStartOpts` and at the top of the file; feels very weird to see these referenced from code above their definitions.

Reply:
Yeah I wasn't totally sure what to name it, but it's essentially the difference between what's passed to `fill_in_opts_from_current_narrowed_view` and what's returned, which is an object with fewer optional values. I'll just say that.